pretix_eth/static/pretix_eth/transaction.js

```diff
@@ -0,0 +1,126 @@
+"use strict";
+
+const BigNumber = window.BigNumber;
+
+const TOKEN_TRANSFER = "0xa9059cbb";
+const DAI_TOKEN_ADDRESS = "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359";
```

Comment:
Am I correct that this moves us from the POA DAI network to the one on mainnet (no problems with thus, just want to be sure I understand).